src/commands/method/hash.ts

```diff
@@ -1,14 +1,14 @@
-const { Command } = require('@oclif/command')
-const { sha3 } = require('ethereumjs-util')
+import { Command } from '@oclif/command'
+import { keccak } from 'ethereumjs-util'
 
-class HashCommand extends Command {
+export default class HashCommand extends Command {
   async run() {
     const { args } = this.parse(HashCommand)
 
     try {
       const { signature } = args
 
-      const hash = sha3(signature)
+      const hash = keccak(signature)
```

Comment:
See release in `ethereumjs-util` for this change
https://github.com/ethereumjs/ethereumjs-util/releases